stack.yaml

```diff
@@ -4,6 +4,8 @@ extra-deps:
   - hspec-junit-formatter-1.1.0.2
   - memcache-0.3.0.1
   - scientist-0.0.0.0
+  - bugsnag-1.0.0.1
+  - bugsnag-wai-1.0.0.1
```

Comment:
Wild guess that I'll need to update the other `stack.yaml` files :smile:

Reply:
All but nightly ;) I just got this into Stackage recently so it should show up in the LTS soon too.